Gemfile.lock

```diff
@@ -1,18 +1,18 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
+    addressable (2.8.3)
```

Comment:
Please omit the Gemfile.lock from the PR, you don't need to commit it.

Reply:
@akshgpt7 can you let me know who are the developer of these specific project?